network/init-neutrond.sh

```diff
@@ -643,6 +643,7 @@ set_genesis_param_jq ".app_state.globalfee.params.bypass_min_fee_msg_types" "$BY
 set_genesis_param proposer_fee                "\"0.25\""                                    # builder(POB)
 set_genesis_param escrow_account_address      "\"$DAO_CONTRACT_ADDRESS_B64\","              # builder(POB)
 set_genesis_param sudo_call_gas_limit                "\"1000000\""                          # contractmanager
+set_genesis_param max_gas                      "\"40000000\""                               # consensus_params
```

Comment:
is this value approved or something? @swelf19 @pr0n00gler

Reply:
The dex module requires max gas to be set to avoid certain DOS scenarios wrt purging expired limit orders. We settled on this number as a sensible max gas per block (ie. high enough to not be hit under normal conditions, low enough to prevent DOS) but we can certainly change it if you guys have an opinion.

Reply:
@swelf19 @sotnikov-s @pr0n00gler

Reply:
Currently we have `max_gas` equals to 30M. I prefer leave it as it is

Reply:
Changed it to 1B since it's required for our integration tests. On mainnet 'max_gas' equals to 30M